README.md

```diff
@@ -125,7 +125,7 @@ $ cd prediction-using-watson-machine-learning
 * Enter a name for the notebook.
 * Optionally, enter a description for the notebook.
 * Under `Notebook URL` provide the following url: https://raw.githubusercontent.com/IBM/prediction-using-watson-machine-learning/master/notebooks/MLpredictor.ipynb
-* Select the Spark runtime with Python 3.5 .
+* Select the Spark runtime with Python 3.5.
```

Comment:
I don't like this syntax. Yes, some might argue that it is correct, but it looks like the patch version is missing for the form `3.5.x`